code/modules/shuttle/marine_dropship.dm

```diff
@@ -140,6 +140,12 @@
 	. = ..()
 	SSshuttle.dropships += src
 
+/obj/docking_port/mobile/marine_dropship/enterTransit()
+	. = ..()
+	if(!.) // it failed in parent
+		return
+	// pull the shuttle from datum/source, and state info from the shuttle itself
+	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_DROPSHIP_TRANSIT)

```

Comment:
You should remove this as it was included in @spookydonut's pr #3563

Reply:
I recommended adding it here before that one is merged.
Personally I think it's better if it's added with functionality already, instead of solo.